lib/modules/platform/github/index.md

```diff
@@ -71,6 +71,7 @@ Any tokens that do not start with `ghs_` (for example tokens from GitHub Enterpr
 !!! note
     The installation tokens expire after 1 hour and need to be regenerated regularly.
     Alternatively as environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
+    There are tools to regenerate the installation token. If you run your self-hosted Renovate Bot in a GitHub Actions workflow, you can use the actions [`jnwng/github-app-installation-token-action`](https://github.com/jnwng/github-app-installation-token-action) or [`tibdex/github-app-token`](https://github.com/tibdex/github-app-token). Other tools are [github-app-installation-token](https://github.com/gagoar/github-app-installation-token) (Node.js libary / CLI) or the [`mshekow/github-app-installation-token`](https://github.com/MShekow/github-app-installation-token) Docker image.
```

Comment:
```suggestion

<!-- prettier-ignore -->
!!! tip "Third-party tools to regenerate installation tokens"
    If you're self-hosting Renovate within a GitHub Actions workflow, then you can use the [`jnwng/github-app-installation-token-action`](https://github.com/jnwng/github-app-installation-token-action) or [`tibdex/github-app-token`](https://github.com/tibdex/github-app-token) actions.
    If you use Node.js/CLI, then you can use the [`github-app-installation-token`](https://github.com/gagoar/github-app-installation-token) package/library.
    If you use Docker, then you can use the [`mshekow/github-app-installation-token`](https://github.com/MShekow/github-app-installation-token) image.
```

Changes:

- Make a new `tip` admonition for the whole block
- Put first sentence in the `tip` title
- Convert rest of text to match the "If you use ..., then you can use ..." pattern
- Use `monospace` for all "package names"
- Fixed typo in word `libary` -> `library`